.github/workflows/publish-github-release.yml

```diff
@@ -0,0 +1,92 @@
+name: Publish GitHub Release as newest
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 3 * * 1-5'
+
+jobs:
+  publish-release:
+    name: Publish GitHub Release
+    runs-on: ubuntu-24.04
+    permissions:
+      contents: write
+      actions: read
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          fetch-depth: 0
+
+      - name: Get latest pre-release name
+        id: get_latest_prerelease
+        run: |
+          latest_release=$(jq -r 'first' <<< $(curl --silent https://api.github.com/repos/bitwarden/ios/releases))
+          is_latest_prerelease=$(jq -r '.prerelease' <<< $latest_release)
+          echo "is_latest_prerelease=$is_latest_prerelease" >> $GITHUB_OUTPUT
```

Comment:
We create the releases as Drafts, not pre-releases; will those be picked up by that logic?

Reply:
It won't. I changed it to pick up the draft releases. Is it the same for android?

Reply:
Yes, that wil be the same for Android!